### PR TITLE
Dish Debugging

### PR DESCRIPTION
**Pull request by ahmadalashtar**
Update status codes
Hide __v from query results
Add image upload route
Update tests
Delegate 'Item not found" to isOwner middleware

#83

### CODE REVIEW

File: src/__tests__/dish.test.js [hunk 1]
```diff
@@ -191,7 +191,7 @@ describe('POST /api/dishes', () => {
       .post('/api/dishes')
       .send({ name: 'hello' })
       .set('Cookie', ownerChefToken);
-    expect(res.status).toBe(422);
+    expect(res.status).toBe(500);
```
Comment (HishamWattar) [inline src/__tests__/dish.test.js:194]: Why did you change it to 500?

**Review by HishamWattar — APPROVED** [review]

**Review by 0Rawan — APPROVED** [review]
